setup.py

```diff
@@ -14,13 +13,19 @@
     url='https://github.com/jacobb/nap',
     long_description=open('README.rst').read(),
     packages=find_packages(),
-    requires=[
-    ],
     zip_safe=False,
-    tests_require=test_requirements,
+    setup_requires=['pytest-runner>=2.0,<3dev'],
+    tests_require=[
```

Comment:
I know I'm a bit late to review this, but in the future, I think I would like to see two separate PRs when focusing on two things - here test updates and flask caching.